deploy/internal/pod-agent.yaml

```diff
@@ -24,10 +24,9 @@ spec:
         - name: CONTAINER_PLATFORM
           value: KUBERNETES
         - name: AGENT_CONFIG
-          valueFrom:
```

Comment:
why did you remove that?

Reply:
Jacky's code is overwriting the entire env not just replacing a value. This declaration has no meaning and is actually misleading. I talked with him and we decided to remove this.